modules/mono/glue/GodotSharp/GodotSharp/Core/Mathf.cs

```diff
@@ -13,14 +19,18 @@ public static partial class Mathf
         /// The circle constant, the circumference of the unit circle in radians.
         /// </summary>
         // 6.2831855f and 6.28318530717959
+#if NET5_0_OR_GREATER
+        public const real_t Tau = MathS.Tau;
+#else
         public const real_t Tau = (real_t)6.2831853071795864769252867666M;
+#endif
```

Comment:
It's not worth it to add complexity for no reason.

```suggestion
        public const real_t Tau = (real_t)6.2831853071795864769252867666M;
```

Reply:
Is it nessesary to be compatibly with netstandard? Well I guess it would be possible to backport, so let's make easy if someone want to make that decision.